app/controllers/hosts_controller.rb

```diff
@@ -421,6 +422,15 @@ def multiple_destroy
   def multiple_build
   end
 
+  def rebuild_tftp
+  end
+
+  def submit_rebuild_tftp
+    @hosts.map(&:interfaces).flatten.select(&:provision).map { |nic| nic.send(:setTFTP) if nic.tftp? }
```

Comment:
1. `@hosts` should be filtered somewhere to only be managed hosts, either in the selection list or perhaps here (though it seems a bit late).
2. I'd also suggest using host.provision_interface or simply `host.interfaces.where(:provision => true)` instead of filtering in-memory.
3. Maybe setTFTP should be a public method, to avoid send?